gamestonk_terminal/loggers.py

```diff
@@ -0,0 +1,181 @@
+"""Terminal helper"""
+__docformat__ = "numpy"
+import logging
+import os
+from pathlib import Path
+import sys
+import time
+import uuid
+
+import git
```

Comment:
We may need to update requirements with this package.

I think we had at some point but I removed it because was giving issues, and manage to go without it.

Reply:
We still have it:

```
$ grep git requirements.txt
gitdb==4.0.9; python_version >= "3.7"
gitpython==3.1.24; python_version >= "3.7"
```